torchgeo/transforms/transforms.py

```diff
@@ -7,107 +7,8 @@
 
 import kornia.augmentation as K
 import torch
-from einops import rearrange
-from kornia.contrib import extract_tensor_patches
 from kornia.geometry import crop_by_indices
-from kornia.geometry.boxes import Boxes
 from torch import Tensor
-from torch.nn.modules import Module
-
-
-# TODO: contribute these to Kornia and delete this file
-class AugmentationSequential(Module):
```

Comment:
Maybe we can have a simple:
```python
from kornia.augmentation import AugmentationSequential

__all__ = ['AugmentationSequential']
```
This will maintain backwards compatibility. Even better would be to find a way to deprecate it like in #2396. Maybe something like:
```python
@deprecated('Use kornia.augmentation.AugmentationSequential instead')
class AugmentationSequential(K.AugmentationSequential):
    ...
```
I'm just worried that this class has been here since TorchGeo 0.1 and although it isn't documented, it's used in all of our builtin datamodules and all of our tutorials.